packages/plugins/experimental/plugin-wnfs/src/components/FilePreview.tsx

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import React, { type FC } from 'react';
+
+import { mx } from '@dxos/react-ui-theme';
+
+export type FilePreviewProps = {
+  type: string;
+  url: string;
+  className?: string;
+};
+
+/**
+ * File/content preview iframe.
+ */
+export const FilePreview: FC<FilePreviewProps> = ({ type, url, className }) => {
```

Comment:
you might be copying one of our antipatterns, since we've done this in the past, but now prefer

FilePreview = ({ ... }: FilePreviewProps) =>